docs/.vitepress/config/index.ts

```diff
@@ -12,6 +13,7 @@ export default defineConfig({
     zh: { label: '简体中文', ...zh },
     pt: { label: 'Português', ...pt },
     ru: { label: 'Русский', ...ru },
-    ko: { label: '한국어', lang: 'ko-KR', link: 'https://vitepress.vuejs.kr/' }
+    ko: { label: '한국어', lang: 'ko-KR', link: 'https://vitepress.vuejs.kr/' },
+    es: { label: 'Español', ...es }
```

Comment:
let's keep built in ones before external translations

```suggestion
    es: { label: 'Español', ...es }.
    ko: { label: '한국어', lang: 'ko-KR', link: 'https://vitepress.vuejs.kr/' }
```